platforms/jvm/ear/src/main/java/org/gradle/plugins/ear/Ear.java

```diff
@@ -199,7 +206,7 @@ private DeploymentDescriptor forceDeploymentDescriptor() {
      * A location for dependency libraries to include in the 'lib' directory of the EAR archive.
      */
     @Internal
-    @ToBeReplacedByLazyProperty(comment = "Should this be lazy?")
+    @NotToBeReplacedByLazyProperty(because = "Nested property")
```

Comment:
❓ If this is a nested property, shouldn't it be marked as `@Nested`? I know `CopySpec`s are handled in a special way, but maybe we can find a better explanation here?

Reply:
I changed to `Read-only CopySpec property`.

I don't think this should be lazy or either `@Nested` since it's CopySpec. 🤔